tools/model_tools/README.md

```diff
@@ -34,20 +34,24 @@ Installation from source is as follows:
 
 1. Install Python (version 3.6 or higher), [setuptools](https://pypi.org/project/setuptools/):
 
+2. Install [openvino-dev](https://pypi.org/project/openvino-dev/) python package of the corresponding version:
+
+```sh
+pip install openvino-dev[caffe,caffe2,onnx,tensorflow2,pytorch,mxnet]
+```
+> **NOTE**: For example, if you are using OMZ Tools for 2021.4.2 then install openvino-dev==2021.4.2.
+
 2. Install the tools with the following command:
 
 ```sh
-python setup.py install
+export OMZ_ROOT=<omz_dir>
```

Comment:
FYI, you should no longer need to do this when using pip 21.3+.

Reply:
@jkamelin have you address this coment?

Reply:
Added note about pip version